docs/user/tutorial/60-admin.rst

```diff
@@ -132,38 +130,36 @@ Let's move a file.
 #.  Once more, use |image_paste| to place the file in the new folder.
 
 
-***************************
 Using images from the Filer
-***************************
+===========================
 
 Now that you have added an image to the Filer, it's easy to use it - and reuse it - whenever you
 need.
 
 .. |close_admin| image:: images/close_admin.png
-   :alt: admin overlay 'close'
+   :alt: the 'close' icon of the admin overlay
```

Comment:
See above - this would become ``the the 'close' icon of the admin overlay icon``.